src/main/java/uk/gov/companieshouse/confirmationstatementapi/controller/PersonsOfSignificantControlController.java

```diff
@@ -43,8 +51,11 @@
 
         try {
             ApiLogger.infoContext(requestId, String.format("Calling PscService to retrieve persons of significant control for companyNumber %s", sanitizedCompanyNumber), logMap);
-            var pscs = pscService.getPSCsFromOracle(sanitizedCompanyNumber);
+            var pscs = pscService.getPSCsFromOracle(submissionId, sanitizedCompanyNumber);
             return ResponseEntity.status(HttpStatus.OK).body(pscs);
+        } catch (SubmissionNotFoundException e) {
```

Comment:
looks like we are doing the submission check in lots of places now, it would be better to create an interceptor that does that in one place, similar to the TransactionInterceptor.